apps/vanilla/src/pages/black-jack/features/player.ts

```diff
@@ -0,0 +1,37 @@
+import { Subscriber } from '../models/subscriber';
+
+import { Publisher } from '../models/publisher';
+
+import { PlayerTurnResult } from '../models/player-turn-result';
+
+const WIN_SCORE = 21;
+
+/** Player. */
+export class Player implements Subscriber<PlayerTurnResult> {
+
+	private readonly diceResults: number[] = [];
```

Comment:
Usually, when representing an array as `readonly`, you should add `readonly` to its type to prevent mutation actions, such as `push`. If you want to use methods like `push`, then remove `readonly` from the variable.

```suggestion
	private readonly diceResults: readonly number[] = [];
```

Reply:
I use readonly here: private readonly diceResults: number[] = []; as I just want diceResult array can not change its whole array but it can still push new element to array.